packages/babel-core/src/config/option-manager.js

```diff
@@ -209,19 +224,28 @@ type BasicDescriptor = {
 /**
  * Load and validate the given config into a set of options, plugins, and presets.
  */
-function loadConfig(config): {
+function loadConfig(
+  config,
+): {
   options: {},
   plugins: Array<BasicDescriptor>,
   presets: Array<BasicDescriptor>,
 } {
   const options = normalizeOptions(config);
 
-  if (config.options.plugins != null && !Array.isArray(config.options.plugins)) {
+  if (
+    config.options.plugins != null &&
+    !Array.isArray(config.options.plugins)
+  ) {
     throw new Error(".plugins should be an array, null, or undefined");
   }
 
   const plugins = (config.options.plugins || []).map((plugin, index) => {
-    const { filepath, value, options } = normalizePair(plugin, loadPlugin, config.dirname);
+    const { filepath, value, options } = normalizePair(
```

Comment:
This is an interesting one where I feel like I'd probably have flattened the `const` binding names, rather than the function args, so the call itself was still all together. e.g.

```
const {
  filepath,
  value,
  options,
} = normalizePair(plugin, loadPlugin, config.dirname);
```